weekly_templates/dc2_pipeline_bps_step1.yaml

```diff
@@ -0,0 +1,42 @@
+pipelineYaml: "$DRP_PIPE_DIR/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step1"
+# qgraphFile: /scratch/brendal4/bps-gen3-dc2/submit/2.2i/runs/test-med-1/w_2021_40/DM-32024/20211005T175202Z/2.2i_runs_test-med-1_w_2021_40_DM-32024_20211005T175202Z.qgraph
```

Comment:
Please remove any comments that are no longer relevant, and trim the ones that are too long (split any paths on multiple lines if needed).